src/test/java/apicurito/tests/utils/slenide/ConfigurationOCPUtils.java

```diff
@@ -0,0 +1,147 @@
+package apicurito.tests.utils.slenide;
+
+import static org.assertj.core.api.Assertions.fail;
+
+import org.yaml.snakeyaml.Yaml;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import apicurito.tests.configuration.TestConfiguration;
+import apicurito.tests.utils.HttpUtils;
+import apicurito.tests.utils.openshift.OpenShiftUtils;
+import io.fabric8.kubernetes.api.model.apps.ReplicaSet;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class ConfigurationOCPUtils {
+
+    public static void waitForOperatorUpdate() {
+        log.info("Waiting for 2 created replica sets");
+        List<ReplicaSet> uiRs = new ArrayList<>();
+
+        //Wait for 2 minutes for replica sets
+        for (int i = 0; i < 12; ++i) {
+            try {
+                Thread.sleep(10000L);
+            } catch (InterruptedException e) {
+                e.printStackTrace();
+            }
+
+            uiRs.clear();
+            List<ReplicaSet> listRs = OpenShiftUtils.getInstance().apps().replicaSets().list().getItems();
+            for (ReplicaSet rs : listRs) {
+                if ("apicurito-service-ui".equals(rs.getMetadata().getOwnerReferences().get(0).getName())) {
+                    uiRs.add(rs);
+                }
+            }
+            if (uiRs.size() == 2) {
+                break;
+            }
+            if (i == 11) {
```

Comment:
Also in case the for loops are in separate method - the fail condition could be made more elegant, outside the for loop.

Reply:
Thats good point. I pulled out loops to separate methods